core/eolearn/core/utilities.py

```diff
@@ -21,6 +22,20 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class LogFileFilter(Filter):
+    """ Filters log messages passed to log file
+    """
+
+    def __init__(self, thread_name, name=''):
+        self.thread_name = thread_name
+        super().__init__(name=name)
```

Comment:
It would be safer to write
```
    def __init__(self, thread_name, *args, **kwargs):
        self.thread_name = thread_name
        super().__init__(*args, **kwargs)
```
Even though current implementation of `Filter` class only has `name` parameter we don't know what will happen in the future Python versions. Hence it's better to just pass any possible parameter forward.